3737/2/commands/checkout_command.rb

```diff
@@ -0,0 +1,60 @@
+require 'fileutils'
+require 'haversine'
+require 'time'
+require_relative '../helpers/photo_helper.rb'
+require_relative '../helpers/location_helper.rb'
+
+# module with checkout command
+module CheckoutCommand
+  def checkout!(*)
+    if User.registered(from['id']) && User[from['id']].in_camp == 'true'
```

Comment:
I'd find user on separate line above and then check
if user && user.in_camp
Will it be more readable?